scalalib/src/publish/ExtraPublish.scala

```diff
@@ -0,0 +1,25 @@
+package mill.scalalib.publish
```

Comment:
I'd like to keep this in package `mill.scalalib` if possible.

Reply:
Thats not possible as it would introduce cyclic dependencies. Perhaps if the PublishModule as such is also moved to scalalib.publish it might work